configs/components/fesom/Subjob_tripy.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+#Home directory and reset of variable
+cd ~
+source ~/.bash_profile
+
+cd ~
+
+# Function to check for py39 envirionmet. If necessary installing and activating.
+#this is just to not trigger errors while activating py39
+#module unload netcdf-c
+#module unload netcdf-fortran
+#Function for activating py39
+activate_env() {
+    cd ~
+    source ~/.bash_profile
+    conda activate py39
+}
+
+cd /work/ab0995/a270275/runconfig
+# Check if conda ist installed
+if ! command -v conda &> /dev/null; then
+    echo "Conda is not installed. Plesa install conda or miniconda"
+    exit 1
+fi
+
+# Überprüfen, ob das Environment "py39" existiert
+if conda info --envs | grep -q "^py39 "; then
+    echo "Py39 is exisiting already. Activating now..."
+    activate_env
+else
+    echo "The Py39 env is not exisiting, installing..."
+    cd~
```

Comment:
This will fail, you need:

```suggestion
    cd ~
```